spec/factories/mentors.rb

```diff
@@ -8,22 +8,19 @@
 #  trn        :string           not null
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
-#  school_id  :uuid             not null
-#
-# Indexes
-#
-#  index_mentors_on_school_id  (school_id)
-#
-# Foreign Keys
-#
-#  fk_rails_...  (school_id => schools.id)
 #
 FactoryBot.define do
   factory :mentor do
-    association :school
-
     first_name { Faker::Name.first_name }
     last_name { Faker::Name.last_name }
     sequence(:trn) { _1 }
   end
+
+  factory :claims_mentor,
+          class: "Claims::Mentor",
+          parent: :mentor
+
+  factory :placements_mentor,
+          class: "Placements::Mentor",
+          parent: :mentor
```

Comment:
Likewise, [this will work](https://thoughtbot.github.io/factory_bot/ref/factory.html#options):

```suggestion
  factory :claims_mentor,
          class: Claims::Mentor,
          parent: :mentor

  factory :placements_mentor,
          class: Placements::Mentor,
          parent: :mentor
```

Reply:
Rubocop will complain about this, but I can skip this rule in this file I think